gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemConfiguration.java

```diff
@@ -353,12 +353,21 @@
   public static final HadoopConfigurationProperty<String> GCS_GRPC_SERVER_ADDRESS =
       new HadoopConfigurationProperty<>("fs.gs.grpc.server.address");
 
-  /** Configuration key for the connect timeout (in millisecond) for gRPC read requests to GCS. */
+  /**
+   * Configuration key for the connection timeout (in millisecond) for gRPC read requests to GCS.
+   */
   public static final HadoopConfigurationProperty<Long> GCS_GRPC_READ_TIMEOUT_MS =
-      new HadoopConfigurationProperty<>("fs.gs.grpc.read.timeout.ms", 20 * 60 * 1000L);
+      new HadoopConfigurationProperty<>("fs.gs.grpc.read.timeout.ms", 30 * 1000L);
+
+  /**
+   * Configuration key for the connection timeout (in millisecond) for gRPC read requests to GCS.
```

Comment:
It is same as above. Should we change documentation of both of `GCS_GRPC_READ_TIMEOUT_MS` and `GCS_GRPC_READ_SPEED_BYTES_PER_SEC` mentioning the formula, actual timeout = base + size/speed...? 

GCS_GRPC_READ_TIMEOUT_MS name also looks misleading now looking like we use this as it is. Are we using dynamic only for actual object get and not for other APIs?

Reply:
We will move the documentation to configuration.md. The code comments here should reflect just the option